install/configure_administrative_access.md

```diff
@@ -101,17 +103,65 @@ on the BMC for node power down or node power up.
 
 See [Configure BMC and Controller Parameters with SCSD](../operations/system_configuration_service/Configure_BMC_and_Controller_Parameters_with_scsd.md).
 
-## 5. Configure non-compute nodes with CFS
+## 5. Set up passwordless SSH
 
-Non-compute Nodes (NCN) need to be configured after booting for administrative access, security, and other
+See [Set up passwordless SSH](../operations/CSM_product_management/Set_Up_Passwordless_SSH.md)
+for the procedure to configure passwordless SSH between management nodes and from management nodes
+to managed nodes.
+
+This procedure sets up resources in Kubernetes (a Kubernetes Secret and ConfigMap) which are later
+applied to the management nodes using CFS node personalization in section
+[7. Configure management nodes with CFS](#7-configure-management-nodes-with-cfs) below.
+
+## 6. Configure the root password and SSH keys in Vault
+
+See [Configure the `root` password and SSH keys in Vault](../operations/CSM_product_management/Configure_the_root_Password_and_SSH_Keys_in_Vault.md)
+for the procedure to configure the `root` password and SSH keys in Vault.
+
+This procedure writes the `root` password hash and SSH keys to Vault which are later
+applied to the management nodes using CFS node personalization in section
+[7. Configure management nodes with CFS](#7-configure-management-nodes-with-cfs) below.
+
+## 7. Configure management nodes with CFS
+
+Management nodes need to be configured after booting for administrative access, security, and other
 purposes. The [Configuration Framework Service (CFS)](../operations/configuration_management/Configuration_Management.md)
-is used to apply post-boot configuration in a decoupled, layered manner. Individual software products including
-CSM provide one or more layers of configuration in a process called "NCN personalization".
+is used to apply post-boot configuration in a decoupled, layered manner. Individual software products
+provide one or more layers included in a CFS configuration. The CFS configuration is applied to components,
+including management nodes, during post-boot node personalization.
+
+The procedure here creates a CFS configuration that contains only the layer provided by the CSM product and
+then applies that configuration to the management nodes.
+
+1. (`ncn-m001#`) Set the variable `CSM_RELEASE` to the CSM release version. For example:
+
+    ```bash
+    export CSM_RELEASE="1.4.0"
```

Comment:
Not a big deal, but no need to export the variable, I don't think -- we're not trying to pass it into any child processes.

```suggestion
    CSM_RELEASE="1.4.0"
```

Reply:
Yeah, sounds right.